integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -1422,6 +1422,7 @@ private[sql] case class DescribeCommandFormatted(
     results ++= Seq(("Table Name : ", relation.tableMeta.carbonTableIdentifier.getTableName, ""))
     results ++= Seq(("CARBON Store Path : ", relation.tableMeta.storePath, ""))
     val carbonTable = relation.tableMeta.carbonTable
+    results ++= Seq(("Table Block Size : ", carbonTable.getBlocksize + " MB", ""))
```

Comment:
in carbonTable, block size is set in MB, it's already readable, so i don't think it need to format